doc/apiv3-documentation.apib

```diff
@@ -2584,6 +2584,12 @@ all properties of the linking work package, including properties added by plug-i
 they can occur as properties or as linked properties. A client has to consult the schema to resolve
 the human readable name of custom fields.*
 
+*Properties that cannot be set directly on parent work packages are inferred from their children instead.
+I.e. start date is the soonest start date from its children whereas the due date is the latest due date from its children.
+The estimated time of a parent work package is the sum of its children estimates. The percentage done of a parent work
+package is the average of the sum of its children percentages done. However, if the percentage done is given by a work
+package's status, then only the status matters and no value is inferred.*
+
```

Comment:
You should probably rephrase this paragraph to be able to use an enumeration, because you basically just name every affected property (which is good).

E.g.
> *Bla bla bla* The following properties can not *bla bla* but they are inferred from their children instead:
> * `startDate` is the soonest *bla bla bla*
> * `dueDate` is the latest *bla bla bla*